include/PR/gbi.h

```diff
@@ -4506,6 +4506,33 @@ _DW({									\
 #define gDPNoOpCloseDisp(pkt, file, line) 	gDma1p(pkt, G_NOOP, file, line, 8)
 #define gDPNoOpTag3(pkt, type, data, n)		gDma1p(pkt, G_NOOP, data, n, type)
 
+// Hopefully I won't forget to ask somebody about this before PRing
```

Comment:
Well, did you remember to ask somebody about this?

Reply:
Dammit.
Yes I did. I need to delete those.
Thanks.